sycl/tools/sycl-ls/sycl-ls.cpp

```diff
@@ -24,6 +24,8 @@
 
 using namespace cl::sycl;
 
+#define NumOfBackends 5
```

Comment:
We shouldn't hardcode it like that. You can use a map, or maybe employ `sycl::backend::all`